src/components/lab/canvas/waves/sun-waves/SunWaves.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Group } from 'react-konva';
+import CloudToAtmosphereWave from './CloudToAtmosphereWave';
+import CloudToGroundWave from './CloudToGroundWave';
+import GroundToAtmosphereWave from './GroundToAtmosphereWave';
+import SunToCloudWave from './SunToCloudWave';
+
+const SunWaves = () => {
+  return (
+    <Group>
+      <SunToCloudWave />
+      <CloudToGroundWave />
+      <CloudToAtmosphereWave />
+      <GroundToAtmosphereWave />
```

Comment:
That definitely works and definitely reduces the number of files! But I have a slight preference for the current setup: (1) Although it's multiple files, it's a bit easier to maintain each Wave since each file is quite small and self-contained, (2) The resulting combined component would be quite large (since we would need to have an array of objects each with `waveBeginsX`, `waveBeginsY`, etc, props mapping onto a `<Wave />`.